modules/migration-debugging-velero-admission-webhooks-ibm-appconnect.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+:_content-type: PROCEDURE
+[id="migration-debugging-velero-admission-webhooks-ibm-appconnect_{context}"]
+= Restoring IBM AppConnect resources
+
+When you back up and restore IBM AppConnect resources, you must use special care when you use Velero to restore. If you have problems restoring these esources, there are several chacks you can run.
+
+.Procedure
+
+. Check if you have any link:https://docs.openshift.com/container-platform/4.9/architecture/admission-plug-ins.html#mutating-admission-plug-in_admission-plug-ins[mutating admission plug-ins] of `kind: MutatingWebhookConfiguration` in the cluster:
++
+[source,terminal]
+----
+oc get MutatingWebhookConfiguration
+----
+
+. Using link:https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#rulewithoperations-v1-admissionregistration-k8s-io[the official Kuberbetes documentation], examine the `.yaml` file of each `kind: MutatingWebhookConfiguration` to ensure that none of its rules block creation of the objects that are experiencing issues.
```

Comment:
You could reverse the structure of this step to state the action first, then provide the reference to the Kubernetes documentation. For example:

```
. Examine the `.yaml` file of each `kind: MutatingWebhookConfiguration` to ensure that none of its rules block creation of the objects that are experiencing issues. For more information, see the link:https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#rulewithoperations-v1-admissionregistration-k8s-io[Kuberbetes documentation].
```